addon/mixins/popup.js

```diff
@@ -63,7 +63,7 @@ export default Mixin.create({
   didCreateLayer() {
     this._super(...arguments);
     if (this.get('hasBlock')) {
-      this._popup = this.L.popup({}, this._layer);
+      this._popup = this.L.popup(this.get('popupOptions'), this._layer);
       this._popup.setContent(this.get('destinationElement'));
       this._layer.bindPopup(this._popup, this.get('popupOptions'));
```

Comment:
Like I've said previously, we're now passing `this.get('popupOptions')` both in the popup creation and on `bindPopup` as well. Is this really necessary? I'd like to understand this.